README.md

```diff
@@ -116,6 +116,7 @@ goofys has been tested with the following non-AWS S3 providers:
 * OpenStack Swift
 * S3Proxy
 * Wasabi
+* Scaleway
```

Comment:
Could you keep this alphabetical?

Reply:
Yeah, my bad. `Object Storage` is the service. I've been using goofys to mount a bucket of 40G+ media files to be streamed via Airsonic. It's been 30 days. Not a problem, stable as rock.